pkg/proxy/register_test.go

```diff
@@ -89,31 +91,24 @@ func createProxyDefinitions() []*Definition {
 }
 
 func createRegisterAndRouter() router.Router {
-	r := createRouter()
+	r := router.NewChiRouter()
 	createRegister(r)
 	return r
 }
 
-func createRouter() router.Router {
-	return router.NewChiRouter()
-}
-
 func createRegister(r router.Router) *Register {
 	var routes []*Route
 
 	definitions := createProxyDefinitions()
 	for _, def := range definitions {
-		routes = append(routes, NewRoute(def))
+		routes = append(routes, NewRoute(def, nil, nil))
 	}
 
-	register := NewRegister(r, createProxy())
+	params := Params{
+		StatsClient: stats.NewStatsdClient("", ""),
```

Comment:
use `stats.NewClient("memory://", "")` in tests - see https://github.com/hellofresh/stats-go#usage-in-unit-tests for details